datafusion/core/tests/sql/explain_analyze.rs

```diff
@@ -779,23 +777,6 @@ async fn csv_explain() {
 
     // Note can't use `assert_batches_eq` as the plan needs to be
     // normalized for filenames and number of cores
-    let expected = vec![
-        vec![
-            "logical_plan",
-            "Projection: #aggregate_test_100.c1\
-             \n  Filter: CAST(#aggregate_test_100.c2 AS Int32) > Int32(10)\
```

Comment:
`CAST(#aggregate_test_100.c2 AS Int32)` will be converted to `#aggregate_test_100.c2`, and the `Int32(10)` will be converted to `Int8(10)`